src/content/developers/docs/intro-to-ether/index.md

```diff
@@ -0,0 +1,91 @@
+
+Ethereum also provides smaller rewards to miners who create ["uncle blocks"](/developers/docs/data-and-analytics/block-explorers/#blocks), which are beyond the scope of this introduction.
+
+## Burning Ether {#burning-ether}
+
+Ether burn occurs in every transaction on Ethereum. When a user pays for their transaction, their [base fee](/developers/docs/gas/#base-fee) is automatically destroyed. [In some blocks](https://etherscan.io/block/12965263), more Ether are burned than minted due to base fee burn.
+
+## Denominations of Ether {#denominations}
+
+Since many transactions on Ethereum are small, Ether has several denominations which may be referenced for smaller amounts.
+
+The denominations below can be found in the [Ethereum Whitepaper](/whitepaper/#currency-and-issuance). Others have been proposed, but Wei, Szabo, and Finney remain the most commonly used denominations. Each is named after a famous member of the cypherpunks, a group of cryptographers whose work influenced today's blockchain space.
+
+| Denomination | Value in Ether   | Background                                                                                                                                                                                                        |
+| ------------ | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Wei          | 10<sup>-18</sup> | Computer scientist whose [b-money](http://www.weidai.com/bmoney.txt) system was a foundational work in blockchain development.                                                                                    |
+| Szabo        | 10<sup>-6</sup>  | Computer scientist who helped [define the concept of smart contracts](https://www.fon.hum.uva.nl/rob/Courses/InformationInSpeech/CDROM/Literature/LOTwinterschool2006/szabo.best.vwh.net/smart_contracts_2.html). |
+| Finney       | 10<sup>-3</sup>  | Digital rights activist and first user of Bitcoin after Satoshi Nakamoto.                                                                                                                                         |
+
+Of these denominations, Wei is particularly important, as it is the smallest possible amount of Ether. As a result, many technical implementations, such as the [Ethereum Yellowpaper](https://ethereum.github.io/yellowpaper/paper.pdf), will base all calculations in Wei.
+
+Giga-wei or "gwei" is often used to describe gas costs on the Ethereum network.
+
+## Transferring Ether {#transferring-ether}
+
+Each [transaction](/developers/docs/transactions/#whats-a-transaction) in the Ethereum blockchain contains a `value` field, which transfers a specified amount of Ether, denominated in Wei, to send from the sender's address to the recipient address.
```

Comment:
We're linking to transactions twice here. I'd keep the second link as it's less distracting to users than the first.

```suggestion
Each transaction in the Ethereum blockchain contains a `value` field, which transfers a specified amount of Ether, denominated in Wei, to send from the sender's address to the recipient address.
```